### ISSUE TITLE
MatcherDefinitions aren't correct for v37.0

### ISSUE THREAD

**Issue by vincentius** [financialforcedev/fflib-apex-mocks#25]
To make the following assertions (Which are currently in `fflib_MatcherDefinitionsTest`) run correctly:
```
	@isTest
	private static void whenIsBlankWithMatchesShouldReturnCorrectResults()
	{
		fflib_IMatcher matcher = new fflib_MatcherDefinitions.StringIsBlank();
		System.assert(!matcher.matches(7));
		System.assert(!matcher.matches('bob'));
		System.assert(matcher.matches(null));
		System.assert(matcher.matches(''));
	}

	@isTest
	private static void whenIsNotBlankWithMatchesShouldReturnCorrectResults()
	{
		fflib_IMatcher matcher = new fflib_MatcherDefinitions.StringIsNotBlank();
		System.assert(!matcher.matches(7));
		System.assert(!matcher.matches(null));
		System.assert(!matcher.matches(''));
		System.assert(matcher.matches('bob'));
	}
```

The matchers must be as follows:
```
       /**
     * StringIsBlank matcher: checks if the supplied argument is a blank String
     */
    public class StringIsBlank implements fflib_IMatcher
    {       
        public Boolean matches(Object arg)
        {
            return arg == NULL || (arg instanceof String ? String.isBlank((String)arg) : false);
        }
    }

    /**
     * StringIsNotBlank matcher: checks if the supplied argument is a non-blank string
     */
    public class StringIsNotBlank implements fflib_IMatcher
    {       
        public Boolean matches(Object arg)
        {
            return arg instanceof String ? !String.isBlank((String)arg) : false;
        }
    }

```

**Comment by frup42** [comment]
You are correct, thanks.
I will make the correction soon!

**Comment by frup42** [comment]
Fixed in #27
